.travis.yml

```
@@ -6,6 +6,7 @@ python:
  - "3.5"
  - "3.6"
  - "pypy"
+ - "pypy3.5"
```

This now points to a more recent version.
```suggestion
 - "pypy3"
```